### ISSUE TITLE
Mobs death animation no longer despawn in lazy chunk in 1.18.2

### ISSUE THREAD

**Issue by Pietr06** [fxmorin/carpet-fixes#78]
**Is your feature request related to a problem? Please describe.**
Im tryng to make a lazy looting guardian farm in 1.18.2, but comedy said me that its not that usefull since now mobs animation no longer despawn in lazy chunk

**Describe the solution you'd like**
Mobs death animation shuld despawn in lazy chunk

**Describe alternatives you've considered**
wdym?

**Additional context**
no, i think thats enogh lol

**Comment by FxMorin** [comment]
Do you mean that the entities should tick in lazy chunks if they are already dead and only their death animation is left?

**Comment by Pietr06** [comment]
Oman, i don't exactly know how they work in 1.17, but since comedy said me that I'm making an issue for that, btw i think if only the dead animation is left, they should get yeeted aka despawn

**Comment by FxMorin** [comment]
I need specifics if you want me to implement it correctly

**Comment by Pietr06** [comment]
Can't u see what changed about death animations deapawning in lazy chunk from 1.17 in the code? I'm not a dev, idk how it actually worked in the past

**Comment by FxMorin** [comment]
Not really, despawn animation is not an animation. It's deathTicks, and its never been related to despawning from my knowledge

**Comment by Pietr06** [comment]
Alr, i asked to comedy more details
